Python Tutorial Exercises(Week 1)/#1.py

```diff
@@ -0,0 +1,7 @@
+#1
```

Comment:
Please start documenting these with their respective exercise question in the format of [script docstrings](https://realpython.com/documenting-python-code/#script-docstrings). For example, this should have something along the lines of:
```
""" Week 1 - Exercise 1
This script takes a number n, prints i^2 for all non-negative numbers that are less than n, where i <= n.
This has the exception for i = 4 and will not print anything.
"""
```